### ISSUE TITLE
Pip Install Error - setup.py: No such file or directory: 'README.md'

### ISSUE THREAD

**Issue by benjaminplanche** [robbmcleod/pyfastnoisesimd#2]
Hello,

The pip install (`pip install pyfastnoisesimd`) is failing. The trace reveals an error thrown in `setup.py` line 52 of the current version, trying to open the `README.md` file to get the `long_description` content. The file seems missing.

Faulty line:
https://github.com/robbmcleod/PyFastNoiseSIMD/blob/58de9c17e8ece7e233205df6eb342e9dee8b4daf/setup.py#L52

Pip install trace:
```
$ pip install pyfastnoisesimd
Collecting pyfastnoisesimd
  Downloading pyfastnoisesimd-0.1.1.tar.gz (42kB)
    100% |████████████████████████████████| 51kB 2.5MB/s
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-dvegy8zz/pyfastnoisesimd/setup.py", line 53, in <module>
        with open('README.md') as fh:
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'
```

**Comment by robbmcleod** [comment]
Hi,

I didn't have a `MANIFEST.in` file for the source distribution.  That's been fixed now in release 0.1.2, as seen in https://github.com/robbmcleod/PyFastNoiseSIMD/commit/9601025aa3e7fbaacfdd8eb490c5c314c915feb2 .

I tested installing from the tarball and it works on my machine. The new version is up on PyPI, would you give it a try?

**Comment by benjaminplanche** [comment]
Perfect, the pip install succeeded. Thanks for the quick fix!
